install_config/upgrading/migrating_etcd.adoc

```diff
@@ -315,12 +304,11 @@ or xref:../../admin_guide/backup_restore.adoc#cluster-restore-single-member-etcd
 . Restart master services; for single master clusters, run:
 +
 ----
-# systemctl start atomic-openshift-master
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 +
 For multiple master clusters, run the following on all masters:
 +
 ----
-# systemctl start atomic-openshift-master-api \
-    atomic-openshift-master-controllers
+# systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
```

Comment:
For single maser or multiple master clusters, the same command is executed, they should be merged together, just like what you did in the above modification.